toolkit/tools/grapher/grapher.go

```diff
@@ -137,15 +137,15 @@ func addUnresolvedPackage(g *pkggraph.PkgGraph, pkgVer *pkgjson.PackageVer) (new
 		return
 	}
 
-	logger.Log.Infof("Adding unresolved node %s\n", newRunNode.FriendlyName())
+	logger.Log.Infof("Adding unresolved node %s.", newRunNode.FriendlyName())
```

Comment:
nit: I don't think the `.` is necessary/desired here

Reply:
I treated this as a regular sentence (although I think I should also surround the node's name in quotes), thus the period. How do you see it?

Reply:
I just thought it was confusing, since it's connected to the node name. Adding quotes would also help

Reply:
Discussed offline, added quotes to make it clearer the period is not part of the node's name.